app/src/main/java/module2/globant/calculator/mvp/view/ActivityView.java

```diff
@@ -8,6 +8,8 @@
 import java.lang.ref.WeakReference;
 
 public class ActivityView {
+
```

Comment:
no need for so much spacing, class code grows pretty fast you will need to place your monitor in vertical position bud